control-plane/connect-inject/peering_dialer_controller.go

```diff
@@ -257,3 +266,43 @@ func (r *PeeringDialerController) deletePeering(ctx context.Context, peerName st
 	}
 	return nil
 }
+
+// requestsForPeeringTokens creates a slice of requests for the peering dialer controller.
+// It enqueues a request for each dialer that needs to be reconciled. It iterates through
+// the list of dialers and creates a request for the dialer that has the same secret as it's
+// secret and that of the updated secret that is being watched.
+// We compare it to the secret in the spec as the resource is dependent on the secret.
+func (r *PeeringDialerController) requestsForPeeringTokens(object client.Object) []reconcile.Request {
+	r.Log.Info("received update for Peering Token Secret", "name", object.GetName())
+
+	// Get the list of all dialers.
+	var dialerList consulv1alpha1.PeeringDialerList
+	if err := r.Client.List(r.Context, &dialerList); err != nil {
+		r.Log.Error(err, "failed to list endpoints")
```

Comment:
is `failed to list endpoints` the correct message here (and also in the acceptor controller) ?